pkg/api/query/querypb/query.proto

```diff
@@ -28,17 +28,18 @@ message QueryRequest {
   int64 time_seconds = 2;
   int64 timeout_seconds = 3;
   int64 max_resolution_seconds = 4;
+  int64 lookback_delta_seconds = 5;
+    
+  repeated string replica_labels = 6;
 
-  repeated string replica_labels = 5;
+  repeated StoreMatchers storeMatchers = 7 [(gogoproto.nullable) = false];
```

Comment:
Isn't this creating a backwards incompatible change in protobufs?

Reply:
Right, I'll check `LookbackDeltaSeconds` to be the last number, thanks 👍

Reply:
done